data/Nigeria/Senate/sources/instructions.json

```diff
@@ -0,0 +1,26 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/nigeria-national-assembly",
+        "query": "SELECT * FROM data WHERE js_position = 'Sen' AND area != ''"
```

Comment:
OK, I've changed that, regenerated the CSV file and regenerated `countries.json`. (`ORDER BY id` was sorting on the IDs as strings, both in my local sqlite DB and via morph's SQL API, so I made the `ORDER BY CAST(id AS int)`.)

I'll make a note to also add an `ORDER BY` to the query for `Nigeria/Representatives`.